vendor/github.com/docker/docker/api/types/container/host_config.go

```diff
@@ -92,15 +92,30 @@ func (n NetworkMode) IsContainer() bool {
 	return len(parts) > 1 && parts[0] == "container"
 }
 
-// ConnectedContainer is the id of the container which network this container is connected to.
-func (n NetworkMode) ConnectedContainer() string {
+// IsNS indicates whether container uses a specific network namespace.
```

Comment:
You can't modify the vendored code.  We should probbaly bight the bullet and pull some of this code into a utils. function.

Reply:
thanks for noticing it, I completely ignored it was under vendor.  I've pushed a new version with the new functions added to the libpod code